Controller/ApiController.php

```diff
@@ -40,9 +36,6 @@ class ApiController
     /** @DI\Inject("jms_translation.config_factory") */
     private $configFactory;
 
-    /** @DI\Inject */
-    private $request;
-
```

Comment:
Same changes are made here: #297 I hope to merge that one before.. I just need a comment form the author.